test/functional/checkObjectExists.js

```diff
@@ -169,6 +169,8 @@ function checkObjectExists(config, name, type, callback) {
     checkObjectExistsODBC(config.transportOptions, object, callback);
   } else if (config.transport === 'ssh') {
     checkObjectExistsSSH(config.transportOptions, object, callback);
+  } else { // we cannot check object existence using the rest transport
```

Comment:
This code style looks very strange to me. Is this preferred by the eslint setup?

Reply:
Yes if we do not have the else leg on the same line as the closing brace the `brace-style` error is shown see https://eslint.org/docs/rules/brace-style

Reply:
I also found this weird at first. We can always disable this rule if we want though.

Reply:
Sorry, I was not referring to the else itself, but the comment on that line instead of on the next line.

Reply:
Oh no that is not a linter issue I will move the comment down a line.

Reply:
As far as bracing styles go, I used to be an adherent to Allman styling, but now I prefer Stroustrup, but I'd probably leave it as-is since that's what the rest of the code uses already.